flume/conf/README.md

```diff
@@ -0,0 +1,54 @@
+# Cygnus configuration Procedure
```

Comment:
Maybe a CHANGES_NEXT_RELEASE entry is needed corresponding to this PR is needed? @vgarciag , pls ask to @frbattid or me if you don't know what the hell is this of CHANGES_NEXT_RELEASE file :)